_topic_map.yml

```diff
@@ -1633,35 +1633,37 @@ Name: Logging
 Dir: logging
 Distros: openshift-enterprise,openshift-origin,openshift-dedicated
 Topics:
-- Name: About cluster logging
+- Name: Release notes
+  File: cluster-logging-release-notes
+- Name: About Logging
```

Comment:
Shouldn't these topic map references be {ProtuctTitle} Logging? This distinguishes the cluster logging feature from the pod logging that comes default with OCP.

Reply:
Discussed w. Christian, @sichvoge, and agreed on "Logging"